.github/CODEOWNERS

```diff
@@ -1,3 +1,6 @@
 /docs/docsite/rst/community/collection_contributors/collection_requirements.rst @ansible/steering-committee
 /docs/docsite/rst/community/steering/* @ansible/steering-committee
 /docs/docsite/rst/roadmap/COLLECTIONS_*.rst @ansible/steering-committee
+/DCO @ansible/community-docs-maintainers
```

Comment:
Would be good to add a comment above this file to say it's owned by RH legal and cannot be changed w/o legal review.